docs/java/features/bapi-and-rfc/overview.md

```diff
@@ -0,0 +1,275 @@
+sidebar_label: Overview
+description: Call a BAPI/RFC Module with SAP Cloud SDK for Java
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+- bapi
+- rfc
+---
+
+## Context ##
+The SAP Cloud SDK provides an easy-to-use API that you can use in your cloud app to develop calls to BAPIs and RFC modules inside SAP S/4HANA. This documentation presents the usage of that API, outlines the technical background and touches upon the relevant boundary conditions.
+
+## What is a BAPI/an RFC Module?
+
+As per the [official SAP documentation](https://help.sap.com/saphelp_46c/helpdata/en/a5/3ec8464ac011d1894e0000e829fbbd/content.htm?no_cache=true), a BAPI is an *precisely defined interface providing access to processes and data in business application systems*. That is, BAPIs allow external systems to integrate with SAP S/4HANA by reading and writing business data through the BAPI as interface. By contrast, an RFC module can also be used for such integrations. However, the recommended approach is to leverage BAPIs wherever possible as BAPIs guarantee more stable and intuitive interfaces.
+
+On the technical level, both BAPIs and RFC modules allow for invocation from external systems through the RFC protocol. As opposed to HTTP, RFC is a binary protocol highly-optimized for fast data transfer between enterprise systems.
+
+You can find more differences between BAPI and RFC module on the programming model. Here is a brief overview:
+- BAPIs have names that **always** start with "BAPI". RFC modules have more flexible naming conventions.
+- BAPIs do not throw exceptions, but leverage return tables to transfer messages to the caller. 
+- BAPIs do not use changing parameters, while RFC modules may do.
+
+To improve readablity, we use the term *BAPI* in the remainder only, but always refer to both BAPIs and RFC modules.
+
+## Technical Overview
+
+SAP offers the [SAP Java Connector](https://support.sap.com/en/product/connectors/jco.html) library, hereafter referred to as *JCo* in short, that allows native access to BAPIs inside SAP systems. That means, one can also invoke BAPIs without using the SAP Cloud SDK by solely leveraging JCo.
+
+However, the SAP Cloud SDK offers a convenient Java API on top of JCo that integrates with other SDK core concepts, such as the destination retrieval on SCP, as well as a nice way to mapping Java entity classes to BAPI input parameters and result sets.
+
+The distinction between SAP Cloud SDK and JCo is important, as the SAP Cloud SDK does *not* package JCo as library with its dependencies. The SDK rather assumes that JCo is available on the JVM classpath. 
+
+On the SCP Neo landscape JCO is automatically provided by the infrastructure, on the SCP Cloud Foundry landscape one must use the SAP Java Build Pack during app deployment. There are other edge cases that we'll cover later in this documentation.
+
+## Call a BAPI in SAP S/4HANA
+
+### Generate a Java project
```

Comment:
I think we should fully omit this. IMO this should be a developer guide, not a tutorial. Otherwise this section would be in every single article of this site.

Reply:
I get the point and see the need of a common agreement. I had the intention we want a fully comprehensive documentation that really helps. When we omit things and let gaps still people need to connect the dots and easily get stuck.